packages/api/tests/serial/endpoints/test-granule-csv.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
```

Comment:
Add a test or two to test that the endpoint is behind authorization. Look at the other endpoint tests and you should be able to pretty much copy/paste.

Reply:
Added some